src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -250,6 +250,24 @@ void GcsServer::DoStart(const GcsInitData &gcs_init_data) {
   gcs_task_manager_->SetUsageStatsClient(usage_stats_client_.get());
   RecordMetrics();
 
+  periodical_runner_.RunFnPeriodically(
+      [this] {
+        if (!gcs_placement_group_manager_) {
+          RAY_LOG_EVERY_MS(INFO, 10000) << "Placement group manager does not exist.";
+          return;
+        }
+        // TODO(rickyx): We should move this, no other callers other than autoscaler
+        // use this info.
+        gcs_resource_manager_->UpdatePlacementGroupLoad(
+            gcs_placement_group_manager_->GetPlacementGroupLoad());
```

Comment:
I think we can remove placement_group_load in gcs_resource_manager since it will get all loads from gcs autoscaler manager now?

Reply:
We can also do that this way: https://github.com/ray-project/ray/pull/40254